bin/atomizer

```diff
@@ -10,12 +10,15 @@
 
 process.title = 'atomizer';
 
-var path = require('path');
-var fs = require('fs');
 var chalk = require('chalk');
+var fs = require('fs');
+var minimatch = require('minimatch');
+var path = require('path');
 var program = require('commander');
+var some = require('lodash/some');
+var union = require('lodash/union');
```

Comment:
@roderickhsiao This doesn't appear to be used?

Reply:
oh I think it got removed from previous PR

Reply:
oh in `parseFiles`